src/components/data/EntitiesContent.js

```diff
@@ -378,7 +459,17 @@ const EntitiesContent = ({
         },
         childrenBefore: ({ entities, columnSettings }) => div({
           style: { display: 'flex', alignItems: 'center', flex: 'none' }
-        }, [
+        }, isDataTabRedesignEnabled() ? [
+          renderExportMenu({ columnSettings }),
+          renderEditMenu(),
+          renderOpenWithMenu(),
+          div({ style: { margin: '0 1.5rem', height: '100%', borderLeft: Style.standardLine } }),
+          div({
+            role: 'status',
+            'aria-atomic': true,
+            style: { marginRight: '0.5rem' }
+          }, [`${selectedLength} row${selectedLength === 1 ? '' : 's'} selected`])
+        ] : [
```

Comment:
Nit: Using the reformatter for lines 460 through 482, I got the below formatting, which reads a bit easier to me.

```javascript
        childrenBefore: ({ entities, columnSettings }) => div({ style: { display: 'flex', alignItems: 'center', flex: 'none' } },
          isDataTabRedesignEnabled() ? [
            renderExportMenu({ columnSettings }),
            renderEditMenu(),
            renderOpenWithMenu(),
            div({ style: { margin: '0 1.5rem', height: '100%', borderLeft: Style.standardLine } }),
            div({
              role: 'status',
              'aria-atomic': true,
              style: { marginRight: '0.5rem' }
            }, [`${selectedLength} row${selectedLength === 1 ? '' : 's'} selected`])
          ] : [
            !snapshotName && renderDownloadButton(columnSettings),
            !_.endsWith('_set', entityKey) && renderCopyButton(entities, columnSettings),
            div({ style: { margin: '0 1.5rem', height: '100%', borderLeft: Style.standardLine } }),
            div({
              role: 'status',
              'aria-atomic': true,
              style: { marginRight: '0.5rem' }
            }, [`${selectedLength} row${selectedLength === 1 ? '' : 's'} selected`]),
            renderSelectedRowsMenu(columnSettings)
          ]),
```

Reply:
What formatter did you use? ESLint doesn't complain about the current format.

I know it's been mentioned before, but it would be great if we could adopt a formatter like [Prettier](https://prettier.io/) and avoid any debate about formatting.

Reply:
I've been using the IntelliJ reformatter. 

I agree with having one that works easily for folks. Let's sort it out offline.

Reply:
Done in [13f83dd](https://github.com/DataBiosphere/terra-ui/pull/2907/commits/13f83dd341c182a2b7cf6a69e31932ecadeab3d2).